### PR TITLE
[Benchmark,CI] Benchmarks in PR

### PR DESCRIPTION
**Pull request by vmoens**

### CODE REVIEW

File: benchmarks/conftest.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```
Comment (apbard) [inline benchmarks/conftest.py:1]: move this file to a separate PR and merge it first. This should solve the error you are seing about missing --rank

Comment (vmoens) [reply benchmarks/conftest.py]: Done thanks

**Review by matteobettini — APPROVED** [review]
LGTM
